tests/integration.rs

```diff
@@ -3391,9 +3391,9 @@ mod common_parallel {
         let mut workload_path = dirs::home_dir().unwrap();
         workload_path.push("workloads");
 
-        let mut vhdx_file_path = workload_path;
-        vhdx_file_path.push(DYNAMIC_VHDX_NAME);
-        let vhdx_path = vhdx_file_path.to_str().unwrap();
+        let vhdx_dir = TempDir::new_in(workload_path.as_path()).unwrap();
+        let vhdx_pathbuf = vhdx_dir.as_path().join(DYNAMIC_VHDX_NAME);
+        let vhdx_path = vhdx_pathbuf.to_str().unwrap();
```

Comment:
Let's make it better by using the tmp folder being created for each test, e.g. reusing `guest.tmp_dir`. Also, I don't think we need the `workload_path` anymore.